template/Dockerfile

```diff
@@ -32,15 +34,27 @@ MAINTAINER Vincent Massol <vincent@massol.net>
 
 # Install LibreOffice + other tools
 # Note that procps is required to get ps which is used by JODConverter to start LibreOffice
+<% if (image == 'debian') print '''\
 RUN apt-get update && \\
   apt-get --no-install-recommends -y install \\
     curl \\
     libreoffice \\
     unzip \\
-    procps \\
-    <% if (db == 'mysql') print 'libmysql-java'
-       if (db == 'postgres') print 'libpostgresql-jdbc-java' %> && \\
-  rm -rf /var/lib/apt/lists/*
+    procps \\''' %>
+    <% if (db == 'mysql' && image == 'debian') print 'libmysql-java && \\'
+       if (db == 'postgres' && image == 'debian') print 'libpostgresql-jdbc-java && \\' %>
+<% if (image == 'debian') print 'rm -rf /var/lib/apt/lists/*' %>
+
+<% if (image == 'alpine') print '''\
+RUN apk update && \
+  apk --no-cache add \
+    curl \
+    libreoffice \
+    unzip \
+    procps
+
+RUN curl -L -o /mysql-connector-java-5.1.34.jar https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.34/mysql-connector-java-5.1.34.jar
```

Comment:
This is potentially a problem because it requires quite some maintenance. Isn't there a way to use the "latest" version?

Reply:
What about postgresql?

Reply:
I searched for the way to add "latest " version or to add through apk but I was unable to find the solution. I just found a bit better way that is to maintain the version through arguments.

https://github.com/LasLabs/docker-alpine-jira/blob/master/Dockerfile

Reply:
It only contains MySQL for now. The changed naming convention of variants in build.gradle supports it.

Reply:
> I searched for the way to add "latest " version or to add through apk but I was unable to find the solution. I just found a bit better way that is to maintain the version through arguments.

Did you commit anything about this?

Reply:
I think we should maintain it with version. Should I commit that?